src/functions/formatedEnum.ts

```diff
@@ -0,0 +1,40 @@
+import { IFunction, IFunctionResult } from '../types';
+import { schema } from './schema';
+
+export const formatedEnum: IFunction = (targetVal, opts, paths, otherValues): void | IFunctionResult[] => {
+  const { enum: enumValues, ...innerSchema } = targetVal;
```

Comment:
Would it make sense to validate the input?
It's quite an assumption that `enumValues` is an array.

Reply:
@P0lip You're right. lodah.isArray would work on this.

However, how should we handle this?
 - throw? This would be caught by the runner and issue a console.warn in stdout
 - bail? And hope this will be caught by an oas related schema rule.

Reply:
I went ahead and chose to ignore and bail out. @P0lip Thoughts?

Reply:
Could perhaps report enum is malformed?